### ISSUE TITLE
Abilities when suited suggestion

### ISSUE THREAD

**Issue by Foxyas** [LtxProgrammer/Changed-Minecraft-Mod#1228]
When the player is suited who should control the abilities is the Transfured one but in the "context" it's happen by the suited player, that means that if the Transfured player who is "suiting" other player try execute the hypnosis ability it should work but the "executer" is the suited player, Basicaly just like the command `execute as Entity` [execute a action as the entity]

**Comment by Foxyas** [comment]
could there be a method to prohibit abilities that cannot be executed when suited btw

**Comment by spacedan228** [comment]
i guess in short you mean that suited players should be able to use abilities

**Comment by spacedan228** [comment]
> > i guess in short you mean that suited players should be able to use abilities
> 
> Yeah but Not the suited player that will have the control,
> 
> What i mean is the "suit" itself

it'd be better if the player had the ability to use abilities, in case the latex is or will go afk and etc., and just to not have to tell them what to do all the time

**Comment by Foxyas** [comment]
> > > i guess in short you mean that suited players should be able to use abilities
> > 
> > Yeah but Not the suited player that will have the control,
> > 
> > What i mean is the "suit" itself
> 
> it'd be better if the suiter (suited player) had the ability to use abilities (if they have control), in case the latex is or will go afk and etc., and just to not have to tell them what to do all the time

Yeah not really because the player [human] wouldn't have any idea how do the Abilities?

Is basically a miracle that you can do every ability of any Transfur instantly after the transfurmation soo why a player that wasn't transfured and just cover by a [transfured player] be able to do any ability?

**Comment by spacedan228** [comment]
minecraft, less code, and its not like they're spells, so itd make sense youd be able to do stuff like switch tf mode

**Comment by KJEntytek303** [comment]
actually i embedded this suggestion in https://github.com/LtxProgrammer/Changed-Minecraft-Mod/issues/1341#issuecomment-3196947908 

<img width="1043" height="737" alt="Image" src="https://github.com/user-attachments/assets/93162662-3366-4a8e-b036-8a254fc7e06a" />
